nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -424,6 +424,14 @@ systemd.services.nginx.serviceConfig.ReadWritePaths = [ "/var/www" ];
      The default output of <literal>buildGoPackage</literal> is now <literal>$out</literal> instead of <literal>$bin</literal>.
    </para>
    </listitem>
+   <listitem>
+    <para>
+     Grafana received a major update to version 7.x. A plugin is now needed for
+     image rendering support, and plugins must now be signed by default. More
+     information can be found
+     <link xlink:href="https://grafana.com/docs/grafana/latest/installation/upgrading/#upgrading-to-v7-0">here</link>.
```

Comment:
Nit: I would consider replacing "here" with "in the Grafana documentation" or similar for better visibility than a 4-character word can provide.